com.zen.browser.flatpak.yml

```diff
@@ -0,0 +1,66 @@
+app-id: com.zen.browser
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+base: org.mozilla.firefox.BaseApp
+base-version: '22.08'
+command: launch-script.sh
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --env=GTK_PATH=/app/lib/gtkmodules
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --socket=x11
```

Comment:
Which firefox version is this based on? 122 and up is wayland default. If that's the case this should be changed to `fallback-x11` and `wayland`.

Otherwise use just `x11`.